inappmessaging/USERGUIDE.md

```diff
@@ -272,6 +272,7 @@ For initial documentation:
 * [SDKCF-2054](https://jira.rakuten-it.com/jira/browse/SDKCF-2054): Converted In-App Messaging to Kotlin
 * [SDKCF-1614](https://jira.rakuten-it.com/jira/browse/SDKCF-1614): Polish the Public API (removed unnecessary public APIs)
 * [SDKCF-2342](https://jira.rakuten-it.com/jira/browse/SDKCF-2342): easy_id targeting
+* [SDKCF-2353](https://jira.rakuten-it.com/jira/browse/SDKCF-2353): Rakuten ID targeting
```

Comment:
We probably shouldn't have links to Jira in the public docs. You can just put the Jira ticker number and remove the link.

Reply:
Removed JIRA links